idd/Energy+.idd.in

```diff
@@ -73923,6 +73923,321 @@ HeatPump:PlantLoop:EIR:Heating,
+       \note Enter the name of an OutdoorAir:Node object.
+  A5 , \field Companion Cooling Heat Pump Name
+       \note The name of the companion HeatPump:AirToWater:FuelFired:Cooling object
+       \note This field is used for a heat pump with switchable heating and cooling mode.
+       \type object-list
+       \object-list HeatPumpAirToWaterFuelFiredCoolingNames
+  A6 , \field Fuel Type
+       \required-field
+       \type choice
+       \key NaturalGas
+       \key Propane
+       \key FuelOilNo1
+       \key FuelOilNo2
+       \key Diesel
+       \key Gasoline
+       \key Coal
+       \key OtherFuel1
+       \key OtherFuel2
+       \default NaturalGas
+       \note Fuel Type (NaturalGas, Propane, Gasoline, Diesel etc.)
+  A7 , \field End-Use Subcategory
+       \type alpha
+       \retaincase
+       \default General
+       \note Any text may be used here to categorize the end-uses in the ABUPS End Uses by Subcategory table.
+  N1 , \field Nominal Heating Capacity
+       \autosizable
+       \minimum> 0
+       \units W
+       \note Nominal Heating Capacity in [W] (autosizeable)
+  N2 , \field Design Flow Rate
+       \autosizable
+       \minimum> 0
+       \units m3/s
+       \note Design Flow Rate in m3/s (autosizeable)
+  N3 , \field Design Supply Temperature
+       \default 60
+       \units C
+       \note Design Supply Temperature in [degree C]
+  N4 , \field Design Temperature Lift
+       \autosizable
+       \default 11.1
+       \units deltaC
+       \note Design Temperature Lift in [degree C]
+  N5 , \field Sizing Factor
+       \minimum 1.0
+       \default 1.0
+       \note Sizing Factor for equipment sizing
+  A8 , \field Flow Mode
+       \required-field
+       \type choice
+       \key NotModulated
+       \key ConstantFlow
+       \key LeavingSetpointModulated
+       \default NotModulated
+       \note Flow Mode for the water side of the fuel-fired absorption heat pump
+  A9 , \field Outdoor Air Temperature Curve Input Variable
+       \required-field
+       \type choice
+       \key DryBulb
+       \key WetBulb
+       \default DryBulb
+       \note Outdoor air temperature curve input variable;
+       \note The options are Outdoor Air Dry Bulb or Wet Bulb temperature for curves
+  A10, \field Water Temperature Curve Input Variable
+       \required-field
+       \type choice
+       \key EnteringCondenser
+       \key LeavingCondenser
+       \default EnteringCondenser
+       \note Water Temperature curve input variable - Condenser Entering or Leaving Water Temperature for curves
+  A11, \field Normalized Capacity Function of Temperature Curve Name
+       \required-field
+       \type object-list
+       \object-list BivariateFunctions
+       \note: CAPFT - Normalized Capacity Function of Temperature Curve name,
+       \note which is a biquadratic curve or a lookup table.
+  A12, \field Fuel Energy Input Ratio Function of Temperature Curve name
```

Comment:
Name

Reply:
Here's a question for nobody in particular. Why do all A field descriptions (and subsequent JSON field names and JSON object field names) end with `Name`? What else is an alpha field going to be? I guess it could be an enumeration, but don't all of those end with `Type`?

Reply:
The "name" is corrected. 

For the other question, I can think of an example is the "end use subcategory", it is just taking in the string given by the user. It could be a `Name` or a `Type`, but does not explicitly end with a `Name` or `Type`.

Reply:
... Name came into use to identify fields that are referencing the Name of a different object. "End Use Subcategory" on the other hand is a free-form string which may or may not be used elsewhere.

Reply:
Assuming no further change is needed after the first addressing.